lib/apiservers/portlayer/restapi/options/options.go

```diff
@@ -17,11 +17,12 @@ package options
 import "time"
 
 type PortLayerOptionsType struct {
-	SDK       string        `long:"sdk" description:"SDK URL or proxy" env:"VC_URL" required:"true"`
-	Cert      string        `long:"cert" description:"Client certificate" env:"VC_CERTIFICATE"`
-	Key       string        `long:"key" description:"Private key file" env:"VC_PRIVATE_KEY"`
-	Insecure  bool          `long:"insecure" default:"true" description:"Skip verification of server certificate" env:"VC_INSECURE"`
-	Keepalive time.Duration `long:"keepalive" default:"5m" description:"Session timeout" env:"VC_KEEPALIVE"`
+	SDK        string        `long:"sdk" description:"SDK URL or proxy" env:"VC_URL" required:"true"`
+	Thumbprint string        `long:"thumbprint" description:"SDK thumbprint" required:"false"`
```

Comment:
if the thumbprint is loaded directly in session.go then you can drop all of the changes related to argument passing